Domesticates the 2nd deck hallway

babydoll opened the pull request:
:cl:
maptweak: The second deck hallway leading from the stairwell and elevators to the engineering foyer has been reconstructed in the style of the typical hallways on the ship.
/:cl:

Purpose of this hallway is general access from the stairs/elevator to the engineering section. Its maintenance-eqsue appearance has deceived some people, so it's time it looks like a real hallway.

Includes lots of warning stripes so the non-maintenence-access-having folks know where to tread.

Map diff will be unhelpful. Video tour: https://streamable.com/1o6wa

Solves https://github.com/Baystation12/Baystation12/issues/24597#issuecomment-466850906

babydoll committed:
Domesticates the 2nd deck hallway

Anticept commented:
You should change that door to engineering to a standard door.

![image](https://user-images.githubusercontent.com/521038/53570249-06cdce80-3b34-11e9-80ec-105ce73e15c6.png)

babydoll committed:
Add in a few new areas for air alarm's sake

Adds air supply setup to the new areas and drone fab

Rotates janitor's room so it faces into the hallway,
instead of maintenance

Reconfigures the area outside janitor's office
to make it two wide

babydoll commented:
Final version pending travis, updated video: https://streamable.com/y2ds5

babydoll commented:
Forgot to add cameras
